.github/workflows/canary-integration-test.yml

```diff
@@ -91,10 +91,22 @@ jobs:
         name: canary
         path: test
 
-    - name: setup tmate session for debugging
-      if: failure()
+    - name: setup tmate session for debugging when event is PR
+      if: failure() && github.event_name == 'pull_request'
       uses: mxschmitt/action-tmate@v3
       timeout-minutes: 120
+      if: |
+        failure() &&
+        (github.event_name == 'push' || startsWith(github.ref, 'refs/tags/v'))
+      uses: mxschmitt/action-tmate@v3
+      timeout-minutes: 30
```

Comment:
@sp98 I tried this it is failing https://github.com/rook/rook/actions/runs/1115983891 so yes multiple `if` not working